apps/vanilla/scripts/getAnime.ts

```diff
@@ -0,0 +1,39 @@
+import { AnimeMapper } from '@js-camp/core/mappers/anime.mapper';
+
+import { AnimeData, AnimeDataFromDto, AnimeRequestData } from './interfaces';
+
+import client from './client';
+
+/**
+ * Function receives data from the server by offset and limit.
+ * @param offset Offset to get the first item in the anime list.
+ * @param limit Maximum number of received anime.
+ * @param ordering The subject of sorting.
+ * @returns Total number of anime series and some anime.
+ */
+async function getAnimeData(offset: number, limit: number, ordering: string): Promise<AnimeDataFromDto> {
```

Comment:
Well, sorry that I didn't make it clear enough. What I meant is not writing "Function" in jsdoc description, I meant thinking that way. 
If we need to explain this function to someone we would say "This function receives data from the server by offset and limit"
But in jsdoc we don't need to write "Function" because we have function keyword. So your description should look like this: "Receives data from the server by offset and limit." 
```suggestion
/**
 * Receives data from the server by offset and limit.
 * @param offset Offset to get the first item in the anime list.
 * @param limit Maximum number of received anime.
 * @param ordering The subject of sorting.
 * @returns Total number of anime series and some anime.
 */
async function getAnimeData(offset: number, limit: number, ordering: string): Promise<AnimeDataFromDto> {
```